src/app.rs

```diff
@@ -511,7 +527,7 @@ pub fn create_app (
                              resp
                          })
                      })
-                     .service(web::resource("/{repo}/{tail:.*}").name("repo")
+                     .service(web::resource("/{tail:.*}").name("repo")
```

Comment:
I worry a bit that this could accidentally override some other path of the API... Especially if we add paths later...

Reply:
I guess we could at least move this rule to the end of the list, and we will never accidentally "replace" the other apis.

Reply:
It's under the `/repo` scope, so I don't know what it could replace.

Reply:
true